src/components/TopicMenu/TopicMenu.js

```diff
@@ -184,8 +188,13 @@ class TopicMenu extends PureComponent {
     const collapsed = isCollapsed || activeTopic.key !== topic.key;
     const isActiveTopic = topic.key === activeTopic.key;
     const isMenuVisibleLayers = (topic.layers || []).find((l) => {
-      return !l.get("hideInLegend");
+      return (
+        !l.get("hideInLegend") &&
+        !l.get("isBaseLayer") &&
```

Comment:
why do we need the `ìsBaseLayer` now ?

Reply:
the little hat ^ was still showing in sandbox TopicMenu, that's why. but I guess I can just add hideInLegend property to the baseLayers instead